packages/wallet/frontend/src/pages/_app.tsx

```diff
@@ -13,6 +18,41 @@ const titilium = Titillium_Web({
 
 export default function App({ Component, pageProps }: AppPropsWithLayout) {
   const getLayout = Component.getLayout ?? ((page) => page)
+  const { toast } = useToast()
+
+  useEffect(() => {
+    let socket: Socket | null = null
+    // Connect to the Socket.IO server
+    socket = io(process.env.NEXT_PUBLIC_BACKEND_URL ?? '', {
```

Comment:
provide config for automatically reconnecting, and maybe fallback option to long-polling